spec/models/profile_spec.rb

```diff
@@ -640,6 +640,16 @@
     end
   end
 
+  describe 'When a profile already has a verified_at timesamp' do
+    it 'does not update the timestamp when #activate is called' do
+      original_timestamp = 1.day.ago
+      profile = user.profiles.create(active: false, verified_at: original_timestamp)
```

Comment:
question: could you use the existing factory with trait `create(:profile, :verified, user: user` (it will have `active: false` already)? If not, could you use or create traits in profile factories?

Reply:
Changed in https://github.com/18F/identity-idp/pull/8612/commits/50e14d85f02f2ca513f1f1bf5f840c07db074019